scripts/poe/tests/translations/matching-placeholders/messages-sw.properties

```diff
@@ -1,3 +1,6 @@
 Accept\ plain-text\ messages = itikia ujumbe wa kawaida
 Number\ of\ facilities = {{number}} mahali
-Number\ of\ form\ types = {{number}} ya aina fomu
\ No newline at end of file
+Number\ of\ form\ types = {{number}} ya aina fomu
+Omitting\ one\ placeholder = {{one}}
+No\ placeholder\ in\ reference\ language = Using {{number}}
```

Comment:
As above, this is the test that I think should fail - we don't know that `{{number}}` is in context.